docs/accordproject-template.md

```diff
@@ -79,7 +79,7 @@ TemplateMark is the markup format in which the text for Accord Project templates
 
 It also provides the ability to specify the document structure (e.g., headings, lists), to highlight certain terms (e.g., in bold or italics), to indicate text which is optional in the agreement, and more.
 
```

Comment:
As a beginner, I was going through the documentation and found that there was a typo in **Markdown Text** hyperlink, hence added a commit to fix it...